README.rst

```diff
@@ -0,0 +1,315 @@
+========================================================
+REANA example - LHCb Rare Charm Decay Search
+========================================================
+
+.. image:: https://img.shields.io/travis/reanahub/reana-demo-lhcb-d2pimumu.svg
+   :target: https://travis-ci.org/reanahub/reana-demo-lhcb-d2pimumu
+
+.. image:: https://badges.gitter.im/Join%20Chat.svg
+   :target: https://gitter.im/reanahub/reana?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge
+
+.. image:: https://img.shields.io/github/license/reanahub/reana-demo-lhcb-d2pimumu.svg
+   :target: https://raw.githubusercontent.com/reanahub/reana-demo-lhcb-d2pimumu/master/LICENSE
+
+
+
+About
+======
+
+This example is based on the
```

Comment:
Please add some introductory text before pointing to where it is based, for example:

This analysis example attempts to reproduce LHCb rare charm decay study published in FIXME. The LHCb produces a lot of ... (see Ana's description) ...  This example is based on (link to Ana's repository).

The goal is that the current repository should be kind of standalone, so containing all the reading, and only then point to the rest.